scripts/submit.ts

```diff
@@ -0,0 +1,20 @@
+import { promises as filesystem } from 'fs'
+import * as path from 'path'
+import { ethers } from 'ethers'
+import dotenv from "dotenv";
+import { runScript } from './utils';
+
+dotenv.config()
+
+async function submitDeploymentTransaction() {
+	const rpcUrl = process.env.RPC
+	const provider = new ethers.providers.JsonRpcProvider(rpcUrl)
+	const chainId = (await provider.getNetwork()).chainId
+	console.log({chainId})
```

Comment:
Maybe a missing `console.log`?

Reply:
You are proposing to remove this? It was meant to provide some feedback what chain is used.

Reply:
Ok, only was a double check to avoid leave custom debug.